topostats/utils.py

```diff
@@ -90,6 +90,27 @@ def update_config(config: dict, args: Union[dict, Namespace]) -> Dict:
     return config
 
 
+def update_plotting_config(plotting_config: dict) -> dict:
```

Comment:
Minor but do you think it might be possible to write a parametrized test for this new function? Could no doubt use the existing `topostats/plotting_config.yaml` and check values are updated correctly by looking at one or two image configurations, say one that should be updated and one that shouldn't.